src/solc_0.7/common/BaseWithStorage/ERC721BaseToken.sol

```diff
@@ -248,17 +223,16 @@ contract ERC721BaseToken is IERC721, WithSuperOperators, WithMetaTransaction {
         }
     }
 
-    /**
-     * @notice Check if the contract supports an interface
-     * 0x01ffc9a7 is ERC-165
-     * 0x80ac58cd is ERC-721
-     * @param id The id of the interface
-     * @return True if the interface is supported
-     */
+    /// @notice Check if the contract supports an interface.
+    /// 0x01ffc9a7 is ERC-165.
+    /// 0x80ac58cd is ERC-721
+    /// @param id The id of the interface.
+    /// @return whether the interface is supported.
```

Comment:
/// @return **W**hether the interface is supported.

Reply:
I'm ok with capitalization here or not... What I went with was no cap in the case where there is NOT a named return param, like this case. 
If this instead returned a bool `isSupported`, then I commented `@return isSupported Whether the interface is supported`
If that's confusing then I'll just go with Caps for all the things

Reply:
Sounds good: I don't mind either way, I thought I noticed some inconsistencies with this but I will look again!